compiler/src/Language/Bond/Parser.hs

```diff
@@ -247,7 +248,11 @@ manySortedBy = manyAccum . insertBy
 field :: Parser Field
 field = makeField <$> attributes <*> ordinal <*> modifier <*> ftype <*> identifier <*> optional default_
   where
-    ordinal = (fromIntegral <$> integer) <* colon <?> "field ordinal"
+    ordinal = fromIntegral <$> (bounded integer) <* colon <?> "field ordinal"
```

Comment:
Instead of `(bounded integer)`, how about making it a `word16` parser?

As for style, please indent lines within `do`. Also commit message title should be no more than about 52 chars (makes for much nicer output from git on command line). Thx.